### PR TITLE
Translation: desktop-client/src/components/budget/BalanceWithCarryover.tsx

### PR DESCRIPTION
**Pull request by nmathey**
Translation support for `desktop-client/src/components/budget/BalanceWithCarryover.tsx`

### CODE REVIEW

File: packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx [hunk 1]
```diff
@@ -135,31 +137,31 @@ export function BalanceWithCarryover({
                   <span style={{ color: theme.noticeText }}>Fully funded</span>
```
Comment (psybers) [inline packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx:137]: Missed some text here.

File: packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx [hunk 2]
```diff
@@ -135,31 +137,31 @@ export function BalanceWithCarryover({
...
                 ) : differenceToGoal > 0 ? (
                   <span style={{ color: theme.noticeText }}>
-                    Overfunded ({format(differenceToGoal, 'financial')})
+                    {t('Overfunded')} ({format(differenceToGoal, 'financial')})
```
Comment (psybers) [inline packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx:140]: Here, I think the amount needs to be part of the translation string.  So something like this:

```suggestion
                    {t('Overfunded ({{amount}})', { amount: format(differenceToGoal, 'financial') })}
```

**Review by psybers — CHANGES_REQUESTED** [review]

File: packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx (hunk 1 above)
Comment (nmathey) [reply packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx]: oups :)

File: packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx (hunk 2 above)
Comment (psybers) [reply packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx:140]: Yes, or maybe in that language parens don't make sense.

**Review by MatissJanis — APPROVED** [review]
Thanks!

**Pull request merged**